iothub/service/src/Common/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -1,56 +1,77 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+
 namespace Microsoft.Azure.Devices.Common.Security
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Globalization;
-    using System.Net;
-    using System.Security.Cryptography;
-    using System.Text;
-
+    /// <summary>
+    /// A helper class to build Shared Access Signature tokens
```

Comment:
Again, you can get rid of the meta "a helper class to". You can just say `Builds Shared Access Signature (SAS) tokens.`